opm/material/fluidmatrixinteractions/EclTwoPhaseMaterialParams.hpp

```diff
@@ -134,7 +134,7 @@ class EclTwoPhaseMaterialParams : public EnsureFinalized
         serializer(*oilWaterParams_);
         serializer(*gasWaterParams_);
     }
-    
+    void setSwl(Scalar val){}
```

Comment:
I did not understand why this was needed? And should we rather throw a logic_error here as well to ensure it is not used?

Reply:
It is need since it is used in the interface for general multiplexer initialization since stone1 and stone2 uses it.